kafka/integration_test.go

```diff
@@ -1559,3 +1559,198 @@ func TestAdminGetMetadata(t *testing.T) {
 	t.Logf("Meta data for admin client: %v\n", metaData)
 
 }
+
+func TestConsumerGetConsumerGroupMetadata(t *testing.T) {
+	if !testconfRead() {
+		t.Skipf("Missing testconf.json")
+	}
+
+	// Prime topic with test messages
+	createTestMessages()
+	producerTest(t, "Priming producer", p0TestMsgs, producerCtrl{silent: true},
+		func(p *Producer, m *Message, drChan chan Event) {
+			p.ProduceChannel() <- m
+		})
+
+	/* Create consumer and subscribe to test-topic
+	 * so that consumer gets registered in broker. */
+	consumerConf := ConfigMap{"bootstrap.servers": testconf.Brokers,
+		"group.id": testconf.GroupID,
+	}
+
+	consumerConf.updateFromTestconf()
+
+	t.Logf("Creating consumer and subscribe to test-topic")
+	c, err := NewConsumer(&consumerConf)
+	if err != nil {
+		t.Fatalf("NewConsumer: %v", err)
+	}
+	defer c.Close()
+
+	c.Subscribe(testconf.Topic, nil)
+
+	// FIXME: wait for topics to become available in metadata
```

Comment:
Dont think this is needed since the consumer is created after the producer is done producing to the topics.